packages/encrypted_storage/lib/src/storage.dart

```diff
@@ -0,0 +1,268 @@
+import 'dart:async';
+
+import 'package:encrypted_storage/src/abstract_storage.dart';
+import 'package:flutter/widgets.dart';
+import 'package:logging/logging.dart';
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+
+/// {@template storage}
+/// Storage (db backend)
+/// {@endtemplate}
+class Storage {
+  /// {@macro storage}
+  Storage();
+
+  late final Database _database;
+  final _log = Logger('EncryptedStorage: Storage');
+
+  /// Init storage
+  Future<void> init([String dbName = 'storage.db']) async {
```

Comment:
May be put dbName as a constant somewhere, same for `encrypted_storage.db`

Reply:
done, sir